skyvern/webeye/utils/dom.py

```diff
@@ -107,6 +108,16 @@ async def is_select2_dropdown(self) -> bool:
             or (tag_name == "input" and "select2-input" in element_class)
         )
 
+    async def is_react_select_dropdown(self) -> bool:
+        tag_name = self.get_tag_name()
+        element_class = await self.get_attr("class")
+        if element_class is None:
+            return False
+
+        return (tag_name == InteractiveElement.INPUT and "select__input" in element_class) or (
```

Comment:
is this also gonna work for the react js in https://bighatbiosciences.pinpointhq.com/en/postings/535636c2-97ac-4efc-ae41-93103507ed64/applications/new?